lib/Config.js

```diff
@@ -43,6 +43,14 @@ class Config {
             this.port = config.port;
         }
 
+        this.listenOn = [];
+        if (config.listenOn !== undefined) {
+            assert(Array.isArray(config.listenOn)
+                && config.listenOn.every(e => typeof e === 'string'),
```

Comment:
Maybe it doesn't matter though since there are many possibilities of accidental string IP addresses.